spec/lib/bamboozled/tabular_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../spec_helper'
+require 'pry'
+
+describe "Employees" do
+
+  before do
+    @client = Bamboozled.client(subdomain:'x', api_key:'x')
```

Comment:
Space missing after colon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.